packages/react-meteor-data/suspense/useSubscribe.ts

```diff
@@ -0,0 +1,64 @@
+import { useEffect } from 'react'
+import { EJSON } from 'meteor/ejson'
+import { Meteor } from 'meteor/meteor'
+import isEqual from 'lodash.isequal'
+import remove from 'lodash.remove'
+
+const cachedSubscriptions: Entry[] = []
+
+interface Entry {
+  params: EJSON[]
+  name: string
+  handle?: Meteor.SubscriptionHandle
+  promise: Promise<void>
+  result?: null
+  error?: unknown
+}
+
+export function useSubscribeSuspense(name: string, ...params: EJSON[]) {
+  const cachedSubscription =
+    cachedSubscriptions.find(x => x.name === name && isEqual(x.params, params))
+
+  useEffect(() =>
+    () => {
+      setTimeout(() => {
+        const cachedSubscription =
+          cachedSubscriptions.find(x => x.name === name && isEqual(x.params, params))
+        if (cachedSubscription != null) {
+          cachedSubscription?.handle?.stop()
+          remove(cachedSubscriptions, x => isEqual(x, cachedSubscription))
```

Comment:
(minor) This could use `===` instead of `isEqual`, as we have a direct reference to the array's element.

Reply:
But `x` and `cachedSubscription` are both objects, that would not work as expected

Reply:
It would, as `find` returns the references to the array element.